galaxyproject/galaxy
# Add harmonize collections tool (or whatever other name)

Add a new tool to synchronize collections. After discussion with IWC it seemed needed for workflows.

## How to test the changes?
(Select all options that apply)
- ?? I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- partially This is a refactoring of components with existing test coverage.
  In theory the tests in the tool wrapper should be sufficient, no?
## License
